sale_report_delivered/reports/sale_report.py

```diff
@@ -56,7 +56,8 @@ class SaleReportDeliverd(models.Model):
     order_id = fields.Many2one("sale.order", "Order", readonly=True)
     picking_id = fields.Many2one("stock.picking", "Picking", readonly=True)
     amount_cost = fields.Float("Amount cost", readonly=True)
-    margin = fields.Float("Margin", readonly=True)
+    margin = fields.Float("Margin delivered", readonly=True)
+    margin_percent = fields.Float(string="Margin delivered(%)", readonly=True)
```

Comment:
Aren't percentage values quite useless (and missleading) in a report? If I get a 50% for a producto in 10 orders, my margin would be 500% as shown in the report :upside_down_face:

Reply:
Look this:
https://github.com/OCA/sale-reporting/pull/134/files#diff-0ea6a8e310c8aedd7d64cd2e9dc00f983471b36f5254176023547b7f678bc2eeR243

Reply:
Oh, nice! :+1: